packages/common/src/services/audius-backend/solana.ts

```diff
@@ -271,6 +271,7 @@ export const pollForBalanceChange = async (
 export type PurchaseContentArgs = {
   id: number
   blocknumber: number
+  extraAmount?: number | BN
```

Comment:
does this need to allow "number"?

Reply:
Yes, mostly because the underlying function also accepts both and I wanted it to be a transparent passthrough.